### PR TITLE
docs: move developer guide into a seperate file

### PR DESCRIPTION
**Pull request by alan-agius4**
Currently, the `readme.md` also contains information on how to build and test the CLI which isn't useful to most users. With this change we move this information into a seperate document.

### CODE REVIEW

**Comment by alan-agius4** [comment]
Preview https://github.com/alan-agius4/angular-cli/tree/developer.md-file

File: README.md [hunk 1]
```diff
@@ -10,44 +10,137 @@
...
+  <br>
+  <img src="/docs/images/angular-cli-logo.png" alt="Angular CLI logo" width="100px" height="108px"/>
+  <br><br>
+  <i>The Angular CLI is a command-line interface tool that you use to initialize, develop, scaffold, 
+    <br>and maintain Angular applications directly from a command shell.</i>
+  <br>
+</p>
 
+<p align="center">
+  <a href="https://cli.angular.io"><strong>cli.angular.io</strong></a>
```
Comment (dgp1130) [inline README.md:25]: I wonder if we should point people at angular.io/cli instead? This subdomain redirects there already and I don't think we have any plans to do anything with it.

File: README.md [hunk 2]
```diff
@@ -41,128 +41,6 @@ Angular DevKit.
 DevKit's goal is to provide a large set of libraries that can be used to manage, develop, deploy and
 analyze your code.
 
-# Getting Started - Local Development
```
Comment (dgp1130) [inline README.md:44]: We should at least link to the new `DEVELOPER.md` doc from the `README`, maybe somewhere under Contributing. That way someone who wants to make a PR can still find the doc, but people who just want to use the CLI will recognize that it isn't for them.

**Review by dgp1130 — APPROVED** [review]
Loving the new look! Very clean and direct.

File: README.md (hunk 1 above)
Comment (alan-agius4) [reply README.md:25]: I don’t have any strong objects. Although in the angular/angular readme cli.angular.io is used.

**Pull request merged**